cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorState.java

```diff
@@ -100,8 +100,8 @@ public void setMetrics(AnomalyMetrics metrics) {
    */
   Map<String, Object> metrics() {
     Map<String, Object> metrics = new HashMap<>(4);
-    metrics.put(MEAN_TIME_BETWEEN_ANOMALIES, _metrics.meanTimeBetweenAnomalies());
-    metrics.put(MEAN_TIME_TO_START_FIX, _metrics.meanTimeToStartFix());
+    metrics.put(MEAN_TIME_BETWEEN_ANOMALIES_MS, _metrics.meanTimeBetweenAnomalies());
```

Comment:
I believe currently `_metrics.meanTimeBetweenAnomalies()` returns mean time per second due to the following line in `AnomalyDetector#anomalyDetectorState()`:
```
meanTimeBetweenAnomalies.put(anomalyType, _anomalyRateByType.get(anomalyType).getMeanRate());
```

We should make the naming and value consistent -- i.e. can we ensure that `meanTimeBetweenAnomalies` map is initialized with ms values rather than sec?

Reply:
good catch!